docs/integrate/status.md

```
@@ -48,6 +48,14 @@ on behalf of [cratedb-examples] and [academy-fundamentals-course].
     <img src="https://img.shields.io/github/actions/workflow/status/crate/cratedb-examples/application-ingestr.yml?branch=main&label=Ingestr" loading="lazy"></a>
 <a href="https://github.com/crate/cratedb-examples/actions/workflows/application-metabase.yml" target="_blank">
     <img src="https://img.shields.io/github/actions/workflow/status/crate/cratedb-examples/application-metabase.yml?branch=main&label=Metabase" loading="lazy"></a>
+<a href="https://github.com/crate/cratedb-examples/actions/workflows/application-pgpool.yml" target="_blank">
+    <img src="https://img.shields.io/github/actions/workflow/status/crate/cratedb-examples/application-pgpool.yml?branch=main&label=Pgpool-II" loading="lazy"></a>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)